edge-modules/api-proxy-module/readme.md

```diff
@@ -99,6 +99,24 @@ When the API Proxy module parses a proxy configuration, it goes through the foll
 
 The parsed configuration is then provided to the nginx reverse proxy.
 
+#### Using the dereferencing feature
```

Comment:
Curious - why do we need this indirection?

Reply:
That allow me to give more option to users. For example:

If I do a config like this:
server_name = ${HOSTNAME}
Then server name is set to the hostname. However, unless they edit the default config file our users cannot override HOSTNAME since it is defined already.
But if I do:
server_name = ${CUSTOM_NAME}
and by default:
${CUSTOM_NAME} = ${HOSTNAME}

Then if user don't put anything, server_name = ${HOSTNAME}
But they can also do ${CUSTOM_NAME} = myname to have
server_name = myname .

Not a critical feature but give flexibility